staging/prometheus-operator/values.yaml

```diff
@@ -60,6 +61,7 @@ mesosphereResources:
     prometheusoverview: true
     scheduler: true
     traefik: true
+    velero: true
```

Comment:
I think for dashboards, it's okay to default them to true since nothing will break if they're installed without the corresponding metrics. Until we figure out how to tie dashboards to addons, we have been defaulting all the dashboards to on for addons that are enabled in knovoy by default, which I believe velero falls under.

Different story for alert rules unfortunately, which may break/misfire if the metrics are missing :)